myhpi/search/templates/search/search.html

```diff
@@ -3,30 +3,43 @@
 
 {% block body_class %}template-searchresults{% endblock %}
 
-{% block title %}Search{% endblock %}
+{% block title %}Suche{% endblock %}
 
 {% block content %}
-    <h1>Search</h1>
-
-    {% include "search/search_field.html" %}
-    <p>
-    {% if search_results %}
-            {% for result in search_results %}
-                    <h4><a href="{% pageurl result %}">{{ result }}</a></h4>
-                    {% if result.search_description %}
-                        {{ result.search_description }}
-                    {% endif %}
-            {% endfor %}
-
-        {% if search_results.has_previous %}
-            <a href="{% url 'search' %}?query={{ search_query|urlencode }}&amp;page={{ search_results.previous_page_number }}">Previous</a>
-        {% endif %}
-
-        {% if search_results.has_next %}
-            <a href="{% url 'search' %}?query={{ search_query|urlencode }}&amp;page={{ search_results.next_page_number }}">Next</a>
-        {% endif %}
-    {% elif search_query %}
-        No results found
+<h1 class="page-title">
+    <span class="underline">
+        Suche
+    </span>
+</h1>
+
+{% include "search/search_field.html" %}
+
+{% if search_results %}
+
+<i class="d-block mb-3 ps-1">
+    {% if search_results|length == 1 %}
+    Ein Ergebnis gefunden.
+    {% else %}
+    {{ search_results|length }} Ergebnisse gefunden.
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#pluralize is your friend :)

Reply:
Good to know! But in this case here, if I want to print "Ein Ergebnis" instead of "1 Ergebnis", this still is the cleaner version, isn't it?